hack/vcluster/partials/main.go

```diff
@@ -3,95 +3,17 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	orderedmap "github.com/wk8/go-ordered-map/v2"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/ghodss/yaml"
 	"github.com/invopop/jsonschema"
 	"github.com/loft-sh/vcluster-docs/hack/platform/util"
 )
 
-// we only generate paths we actually need
```

Comment:
I wonder what was the reason for us to keep this slice manual? Is there a scenario where we would _not_ want to generate full partials? cc @FabianKramm